admin/RELEASE.md

```diff
@@ -43,6 +43,7 @@ git clone git@github.com:codeigniter4/userguide.git
 * Set the date in **user_guide_src/source/changelogs/{version}.rst** to format `Release Date: January 31, 2021`
 * Create a new changelog for the next version at **user_guide_src/source/changelogs/{next_version}.rst** and add it to **index.rst**
 * Create **user_guide_src/source/installation/upgrade_{ver}.rst**, fill in the "All Changes" section, and add it to **upgrading.rst**
+    * git diff --name-status master
```

Comment:
It looks like we could use a path spec exclusion here:
```suggestion
    * git diff --name-status master -- . ':!system'
```
https://stackoverflow.com/questions/5685007/making-git-log-ignore-changes-for-certain-paths/21079437#21079437